### DISCUSSION TITLE
How to read 1Z-alkenylglycerophosphoethanolamines [GP0207] in LipidLynxX

### DISCUSSION THREAD

**Discussion by JauntyJJS** [SysMedOs/LipidLynxX#23] [Q&A] [answered]
Hi,

Not sure if this is missed out or I choose the wrong place to ask a question but I tried to ask a question in [this Github issue](https://github.com/SysMedOs/LipidLynxX/issues/22).

Could anyone help me in this ?

Thank you.

**Comment by ZhixuNi** [comment, marked as answer]
Thanks for your feedback!
I've just replied you in the GitHub issues.
